hw1/02-palindrome.html

```diff
@@ -6,20 +6,26 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body style="background-color: blue">
     <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+      <section
+        style="background-color: white"
+        class="position-absolute top-50 start-50 translate-middle p-3 w-50"
```

Comment:
These bootstrap classes are great & add a lot of margins which is great for desktop, but can squish everything a bit on mobile. If possible would recommend using a "container" class to wrap this form or add a "max-width" style rule to keep the form at a smaller width but allow it to be more visible on mobile.

Reply:
Great point , I noticed that the size would squish on a smaller screen, and I was going to work on it, but I forgot. Thank you for reminding me!